Makefile

```diff
@@ -64,6 +64,9 @@ include ./common/common_subdir.mk
 .PHONY: linux_load
 .PHONY: linux_unload
 .PHONY: linux_clean
+.PHONY: osx_load
+.PHONY: osx_unload
+.PHONY: osx_clean
```

Comment:
please remove these. these will be removed for linux as well, and will be changed to driver_load, driver_unload at some point with the goal that is the makefile can do it, it will detect the OS and run the proper commands for you